pkgs/top-level/python-packages.nix

```diff
@@ -674,6 +674,8 @@ in {
 
   flufl_lock = callPackage ../development/python-modules/flufl/lock.nix { };
 
+  fluidasserts = callPackage ../development/python-modules/fluidasserts { };
```

Comment:
if this is just meant to be used an application, you can move it out of python-modules.

python-modules is really just meant for other packages to import the package. Applications don't need to be there.

Reply:
it's used more as a package than a application, but anyway it's used in both ways

Reply:
alright then, I'll make a new review